sylvia/src/types.rs

```diff
@@ -1,5 +1,10 @@
 use cosmwasm_std::{Deps, DepsMut, Env, MessageInfo};
 
+pub struct ReplyCtx<'a> {
+    pub deps: DepsMut<'a>,
+    pub env: Env,
+}
+
```

Comment:
could be type alias of MigrateCtx

Reply:
I was thinking about that, but not sure about aliasing. I am not sure if there are any drawbacks to that.I don't see any to be honest.